composer.json

```diff
@@ -25,14 +25,13 @@
         }
     },
     "require": {
-        "php": "^5.6 || ^7.0",
+        "php": "^7.3 || ~8.0.0",
         "ext-mbstring": "*",
-        "laminas/laminas-zendframework-bridge": "^1.0",
-        "paragonie/random_compat": "^2.0.11 || 9.99.99"
```

Comment:
https://github.com/paragonie/random_compat is a PHP 5 polyfill and so is no longer needed, correct?

Reply:
Right!